<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter produits</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>581</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ajout">
     <attribute name="title">
      <string>ajouter un produit</string>
     </attribute>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>561</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_code">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_type">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_date">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="valider">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>190</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>190</y>
         <width>351</width>
         <height>141</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Downloads/projectttt__2_-removebg-preview.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>280</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: italic 12pt &quot;8514oem&quot;;
color: rgb(85, 170, 0);</string>
       </property>
       <property name="text">
        <string>Green Renew App</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_date_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_date_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>240</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>quantity</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>prix</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>afficher un produit </string>
     </attribute>
     <widget class="QTableView" name="tap_produit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>531</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>supprimer un produit </string>
     </attribute>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);</string>
      </property>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;8514oem&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_code_supp">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>190</y>
         <width>351</width>
         <height>141</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../Downloads/projectttt__2_-removebg-preview.png</pixmap>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>280</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: italic 12pt &quot;8514oem&quot;;
color: rgb(85, 170, 0);</string>
       </property>
       <property name="text">
        <string>Green Renew App</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
